cirq-core/cirq/protocols/control_key_protocol.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Protocol for object that have control keys."""
+
+from typing import AbstractSet, Any, Iterable
+
+from typing_extensions import Protocol
+
+from cirq._doc import doc_private
+
+
+class SupportsControlKey(Protocol):
```

Comment:
Clean up docstrings in this file - it looks like there was a copy & replace from measurement keys, but the meaning doesn't line up.